<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chat</class>
 <widget class="QWidget" name="chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_send">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>6</y>
     <width>281</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>0</y>
     <width>141</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ip">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>210</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_send">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>51</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>send</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_dis">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>71</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>disconn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_con">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>250</y>
     <width>51</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>conn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>71</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>chatxt</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
